embedded-graphics/src/style/text_style.rs

```diff
@@ -1,12 +1,19 @@
-use crate::fonts::Font;
-use crate::pixelcolor::PixelColor;
+use crate::{fonts::Font, pixelcolor::PixelColor};
 
 /// Style properties for texts.
 ///
 /// `TextStyle` can be applied to a [`Text`] object to define how a text is drawn.
 ///
+/// Because `TextStyle` has the [`non_exhaustive` attribute], it cannot be created using a struct
+/// expression. To create a `TextStyle`, the [`text_style!()` macro] or [`TextStyleBuilder` builder]
```

Comment:
```suggestion
/// literal. To create a `TextStyle`, the [`text_style!`] macro or [`TextStyleBuilder`] builder
```
Or is builder redundant in this case?
```suggestion
/// literal. To create a `TextStyle`, the [`text_style!`] macro or [`TextStyleBuilder`]
```